src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/NotificationPicker.java

```diff
@@ -107,23 +107,24 @@ public NotificationPicker(final IContentManager contentManager, @Named(CONTENT_I
                 // skip expired notifications
                 continue;
             }
-            
-
-            if (null == record) {
-                // either the use hasn't responded to the notification before...
-                resultsToReturn.add(c);
-            } else if (record.getStatus().equals(NotificationStatus.POSTPONED)) {
-                // or they have but they postponed it...
-                Calendar postPoneExpiry = Calendar.getInstance();
-                postPoneExpiry.setTime(record.getCreated());
-                postPoneExpiry.add(Calendar.SECOND, Constants.NUMBER_SECONDS_IN_ONE_DAY);
 
-                if (new Date().after(postPoneExpiry.getTime())) {
+            if (c.getTags().iterator().next().equals(user.getRole().toString().toLowerCase())) {
```

Comment:
This is definitely not the right or best way to do this. What happens if it is tagged both `student` and `teacher`? Or has another tag before the role tag? Or is not tagged and should thus be shown to everyone?

I think 
```java
            if (c.getTags() != null && !c.getTags().contains(user.getRole().name().toLowerCase())) {
                // Skip irrelevant notifications
                continue;
            }
```
after the skip expired ones will do these things?